src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/bolts/FlowRerouteQueueBolt.java

```diff
@@ -106,15 +110,15 @@ protected void init() {
     public void declareOutputFields(OutputFieldsDeclarer declarer) {
         super.declareOutputFields(declarer);
         declarer.declareStream(TimeWindowBolt.STREAM_TIME_WINDOW_EVENT_ID, new Fields(FIELD_ID_CONTEXT));
-        declarer.declareStream(STREAM_FLOWHS_ID, RerouteTopology.KAFKA_FIELDS);
+        declarer.declareStream(STREAM_OPERATION_QUEUE_ID, FIELDS_OPERATION_QUEUE);
         declarer.declareStream(STREAM_NORTHBOUND_ID, RerouteTopology.KAFKA_FIELDS);
     }
 
     @Override
     public void sendRerouteRequest(String correlationId, FlowRerouteRequest request) {
         log.info("Send reroute request {} with correlationId {}", request, correlationId);
-        getOutput().emit(STREAM_FLOWHS_ID, getCurrentTuple(),
-                new Values(correlationId, new CommandMessage(request, System.currentTimeMillis(), correlationId)));
+        emit(STREAM_OPERATION_QUEUE_ID, getCurrentTuple(),
```

Comment:
Do we really want to use anchored emit here (`getCurrentTuple()`)?

Reply:
Anchor has been removed.